src/main/java/com/google/maps/DirectionsApi.java

```diff
@@ -30,21 +30,20 @@
  * text strings (e.g. "Chicago, IL" or "Darwin, NT, Australia") or as latitude/longitude
  * coordinates. The Directions API can return multi-part directions using a series of waypoints.
  *
- * <p>See <a href="https://developers.google.com/maps/documentation/directions/intro">documentation</a>.
```

Comment:
It's not a big deal but please note that this file has nothing to do with the change.